go/l2geth-exporter/main.go

```diff
@@ -51,7 +56,8 @@ func main() {
 		</body>
 		</html>`))
 	})
-	go getCTCTotalElements(ctcAddress, client)
+	go getCTCTotalElements(ctcAddress, "ctc", client)
+	go getSCCTotalElements(sccAddress, "scc", client)
```

Comment:
Is it safe to use the same client in both go routines?

Reply:
I believe it shouldn't be a problem

Reply:
the client should be goroutine-safe.

Reply:
k, thanks